<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>device_curtain</class>
 <widget class="QWidget" name="device_curtain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_background">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="widget_pic" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>227</width>
        <height>184</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_curtain">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>227</width>
         <height>184</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 85, 195);</string>
       </property>
       <property name="text">
        <string>上层图</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="lbl_data">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>500</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_text_curtain">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>百叶窗</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_down">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>500</y>
        <width>46</width>
        <height>44</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_onoff">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_up">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>46</width>
        <height>44</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_lan">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>窗栏</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
